002-quarkus-all-extensions/pom.xml

```diff
@@ -336,15 +297,11 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-spring-data-jpa</artifactId>
         </dependency>
-        <!--        Caused by: java.lang.RuntimeException: java.lang.ClassCastException: io.sentry.servlet.SentryServletContainerInitializer cannot be cast to javax.servlet.ServletContainerInitializer-->
-        <!--      <dependency>-->
-        <!--        <groupId>io.quarkus</groupId>-->
-        <!--        <artifactId>quarkus-logging-sentry</artifactId>-->
-        <!--      </dependency>-->
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-azure-functions-http</artifactId>
-        </dependency>
+        <!--        Disabled because of: Multiple producers of item class io.quarkus.vertx.http.deployment.RequireVirtualHttpBuildItem-->
```

Comment:
sounds like a bug, which extension does it conflict it with?

Reply:
Removed because of conflict with amazon http lambda extension